tests/ui/doc_unsafe.rs

```diff
@@ -0,0 +1,19 @@
+/// This is not sufficiently documented
+pub unsafe fn destroy_the_planet() {
+    unimplemented!();
+}
+
+/// This one is
+///
+/// # Safety
+///
+/// This function shouldn't be called unless the horsemen are ready
+pub unsafe fn apocalypse(universe: &mut ()) {
+    unimplemented!();
+}
+
```

Comment:
Could you also add a test that the lint doesn't trigger on non-async functions?

Reply:
You mean an async function? How would that work?

Reply:
ah sorry, I meant adding an example for a non-unsafe function :D